### ISSUE TITLE
question: custom fields

### ISSUE THREAD

**Issue by dantrevino** [enesser/vCards-js#44]
I don't see any reference to setting custom field (X-FOO-BAR: something).  Is there a way to do this that I'm missing?

tia,
dan

**Comment by enesser** [comment]
No, but an excellent suggestion. Only the most used extension fields are supported right now. I may get to this soon but also welcome any PR that would have extension fields in the format of something like: `vCard.extensions['X-FOO-BAR'] = 'something';`
